scss/input/_layout.scss

```diff
@@ -191,6 +199,122 @@
         }
     }
 
+    // Forms
+    $form-spacer: $padding-x * 2 !default;
+
+    .k-form,
+    .k-form-inline {
+        font-size: $font-size;
+        line-height: $line-height;
+        padding: $form-spacer;
+
+        fieldset {
```

Comment:
I could not find any references for `.k-fieldset` in the sources of Kendo UI for jQuery or Angular. Do we use it anywhere I'm missing or should we erase it?